README.md

```diff
@@ -121,26 +122,23 @@ In the project directory, you can run:
 
 #### `npm start`
 
-Runs the app in the development mode.
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Runs the app in the development mode and opens a new electron window.
 
 The page will reload if you make edits.
 You will also see any lint errors in the console.
 
 #### `npm test`
 
 Launches the test runner in the interactive watch mode.
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
 #### `npm run build`
 
 Builds the app for production to the `build` folder.
 It correctly bundles React in production mode and optimizes the build for the best performance.
 
 The build is minified and the filenames include the hashes.
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Also builds an electron app installer for linux in windows in the folder `/dist`.
```

Comment:
Humm `for linux in windows in` Do you mean for WSL?

Reply:
maybe `and` windows?

Reply:
oops, meant to write `linux and windows`